WebRoot/js/zimbraAdmin/domains/model/ZaDomain.js

```diff
@@ -741,6 +744,16 @@ function(tmpObj, newDomain) {
 		attr.setAttribute("n", ZaDomain.A_zimbraFeatureCalendarReminderDeviceEmailEnabled);
 	}
 
+	for (var i = 0; i < ZaDomain.extendedAttrs.length; i++) {
+		var attrName = ZaDomain.extendedAttrs[i];
+		var attrValue = tmpObj.attrs[attrName];
+	  
+		if (attrValue !== undefined) {
```

Comment:
Isn't there any case that it is empty string or null? If it's empty or null, the attribute does not need to be added to the soapDoc, I think.

Reply:
That's correct, May the changes.